app/models/expert.rb

```diff
@@ -2,4 +2,8 @@ class Expert < ApplicationRecord
   belongs_to :user
   has_many :expert_fields
   has_many :fields, through: :expert_fields
+  has_many :requests
+
```

Comment:
I think we have to add has many reviews through .......
because in the reviews, we will need to be able to use expert.reviews to get all the review of one expert